solutions/agents/variables.tf

```diff
@@ -41,82 +41,82 @@ variable "cluster_config_endpoint_type" {
 
 variable "log_analysis_enabled" {
   type        = bool
-  description = "Deploy IBM Cloud Logging agent if set as true."
+  description = "Whether to deploy the IBM Cloud logging agent."
   default     = true
 }
 
 
 variable "log_analysis_agent_tags" {
   type        = list(string)
-  description = "List of tags to associate to all log records that the agent collects so that you can identify the agent's data quicker in the logging UI. NOTE: Use the 'log_analysis_add_cluster_name' variable to add the cluster name as a tag."
+  description = "The list of tags to associate with all log records collected by the agent so that you can quickly identify the agent’s data in the logging UI. To add the cluster name as a tag, use the `log_analysis_add_cluster_name` variable."
   default     = []
   nullable    = false
 }
 
 variable "log_analysis_add_cluster_name" {
   type        = bool
-  description = "If true, configure the log analysis agent to attach a tag containing the cluster name to all log messages."
+  description = "Whether to attach the cluster name to log messages. Set to `true` to configure the IBM Log Analysis agent to tag all log messages with the name."
   default     = true
 }
 
 variable "log_analysis_ingestion_key" {
   type        = string
-  description = "Ingestion key for the IBM Cloud Logging agent to communicate with the instance."
+  description = "The ingestion key that is used by the IBM Cloud logging agent to communicate with the instance."
   sensitive   = true
   default     = null
 }
 
 variable "log_analysis_secret_name" {
   type        = string
-  description = "The name of the secret which will store the ingestion key. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the secret that stores the ingestion key. If a prefix input variable is specified, the secret name is prefixed to the value in the `<prefix>-<name>` format."
   default     = "logdna-agent"
   nullable    = false
 }
 
 variable "log_analysis_instance_region" {
   type        = string
-  description = "The region name where the IBM Log Analysis instance is created. Used to construct the ingestion endpoint."
+  description = "The name of the region where the IBM Log Analysis instance is created. This name is used to construct the ingestion endpoint."
   default     = null
 }
 
 variable "log_analysis_endpoint_type" {
   type        = string
-  description = "Specify the IBM Log Analysis instance endpoint type (public or private) to use. Used to construct the ingestion endpoint."
+  description = "Specify the IBM Log Analysis instance endpoint type (`public` or `private`) to use to construct the ingestion endpoint. Possible values: `string`."
```

Comment:
Shouldn't possible values be `public` or `private`

Reply:
@iamar7 you're right. 

description = "Specify the IBM Log Analysis instance endpoint type to use to construct the ingestion endpoint. Possible values: `public` or `private`."

Reply:
```suggestion
  description = "Specify the IBM Log Analysis instance endpoint type to use to construct the ingestion endpoint. Possible values: `public` or `private`."
```